### DISCUSSION TITLE
Permalinks in Mardkdown, how do I reference them across projects?

### DISCUSSION THREAD

**Discussion by Stzx** [community/community#61520] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body
<del>Permanent link seems to work only in linked project?</del>

EDIT:
How can I make the link below achieve the same effect as above?

https://github.com/community/community/blob/b8e7de4032a294e6978ab0b2f820742314b25227/CODE_OF_CONDUCT.md?plain=1#L3-L8

https://github.com/community/OSPO/blob/761b31fd95adf2ac2f6525b12f2a17170fa3c060/README.md?plain=1#L13-L15

**Comment by leafstrat** [comment]
[Not sure what you mean](https://github.com/doocs/doocs.github.io/blob/e0834f42e24aa25cc508d5b150327d7bc812b315/README.md?plain=1#L1)

```
[Not sure what you mean](https://github.com/doocs/doocs.github.io/blob/e0834f42e24aa25cc508d5b150327d7bc812b315/README.md?plain=1#L1)
```
